src/lib.rs

```diff
@@ -196,6 +196,7 @@ trait Context: Debug + Send + Sized + Sync {
 
     fn device_features(&self, device: &Self::DeviceId) -> Features;
     fn device_limits(&self, device: &Self::DeviceId) -> Limits;
+    fn device_get_swap_chain_preferred_format(&self, device: &Self::DeviceId) -> TextureFormat;
```

Comment:
we still need an exposed method of the `Device` to access this

Reply:
Resolved on https://github.com/gfx-rs/wgpu-rs/pull/693/commits/f343c2384096c4a855f27a4232b9120441fe9af2